tools/builder.js

```diff
@@ -2,7 +2,7 @@
 
 "use strict";
 const colors = require("colors");
-const fsp = require("fs-extra");
```

Comment:
We can remove `fs-extra` from package.json now, right?

Reply:
`copydeps` still uses `fs-extra` for its own utility functions, but maybe we could cook our own ones.

Reply:
I checked. `cp` is being used only for files (not directories), so we can use the native one easily. And we have `fs.rmdir` and `fs.unlink` for `fsp.remove` 🎉

Reply:
@saschanaz Do you plan to fix above here or separate PR?

Reply:
Hmm, maybe a separate one.